ng-client/src/app/app.module.ts

```diff
@@ -1,29 +1,18 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { HttpModule } from '@angular/http';
-import { FormsModule } from '@angular/forms';
+import { NgModule } from '@angular/core';
```

Comment:
I was wondering what that CUSTOM_ELEMENTS_SCHEMA was even doing. I think that's for Polymer elements or something :/